WebRoot/js/zimbraMail/prefs/view/ZmShortcutsPage.js

```diff
@@ -107,7 +108,8 @@ function() {
 
 	var html = [], i = 0;
 	html[i++] = "<html><head>";
-	html[i++] = "<link href='" + appContextPath + "/css/zm.css' rel='stylesheet' type='text/css' />";
+	//adding skin query param in the css being loaded to load compiled stylesheet.
+	html[i++] = "<link href='" + appContextPath + "/css/zm.css?skin="+ window.skin.hints.name +"' rel='stylesheet' type='text/css' />";
```

Comment:
sure, replacing it with appCtxt.getSkinHint("name"): ae27f22117be80e8f1be2d5f058ce51dafc87bcc